WalletWasabi.Fluent/Helpers/TextHelpers.cs

```diff
@@ -104,11 +104,11 @@ public static string FormatPercentageDiff(double n)
 		if (Math.Abs(withFriendlyDecimals) < precision)
 		{
 			var threshold = n > 0 ? "+" + precision : "-" + precision;
-			return "less than " + threshold.ToString(CultureInfo.InvariantCulture) + "%";
+			return $"{Lang.Resources.Sentences_less_than} " + threshold.ToString(CultureInfo.InvariantCulture) + "%";
```

Comment:
We will have a lot of fun translating this to Arabic.

Reply:
This translation implementation is in fact 50% of the work it would require to translate to non-western languages. There are many other considerations that this PR doesn't handle such as number culture and all this kind of stuff.

Because translating to languages we don't know will not happen anytime soon, I suggest to ignore it

Reply:
Using Ubuntu latest doesn't make work less interesting but rather increases it's core values for equipping engineers with much needed skills